src/me/eglp/gv2/util/base/guild/reminder/ReminderRepetitionEnum.java

```diff
@@ -0,0 +1,49 @@
+package me.eglp.gv2.util.base.guild.reminder;
+
+/**
+ * This is a reminders helper Enum, to signal, how long to wait, between
+ * repetitions of the reminder.
+ * 
+ * @author The Arrayser
+ * @date Mon Mar 27 19:49:22 2023
+ */
+
+public enum ReminderRepetitionEnum {
+	YEARLY("Yearly", 1, 0, 0, 0), MONTHLY("Monthly", 0, 1, 0, 0), WEEKLY("Weekly", 0, 0, 1, 0),
```

Comment:
More destroyed formatting from the formatter

Reply:
Formatting does not hinder the functionality, I think

Reply:
But it impedes readability and thus maintainability